en/index.md

```diff
@@ -322,6 +323,118 @@ permitting users or customers to conduct certain activities.
 It can be tempting to use flowery, official-sounding words
 rather than plain English. Try to keep it simple.
 
+## Interacting with UI elements
+
+### **Screenshots**
+
+**Screenshots should be used sparingly**
+
+A screenshot is not a replacement for clear descriptions in documentation. If an image is well described, a screenshot shouldn’t be necessary in many situations, and including many screenshots can clutter the documentation.
+
+Screenshots also can’t be translated, so they aren’t as accessible to non-native English users or those using translated documentation. Additionally, those using screen-readers won’t be able to access the screenshots without alt-text.
+
+### **Using UI elements as the English words**
+
+**Don’t use UI elements like they’re English words**
+
+For example:
+
+- **Use**: When you’re finished, click **Save** to save your settings.
+- **Don’t use**: When you’re finished, **Save** your settings.
+
+Using the UI text as English words is less clear and may not transfer well to certain translated versions.
+
+### **Click vs. Tap vs. Select**
+
+**Use “Click” for buttons that you click on (or “Tap” if the product is primarily for mobile devices)**
+
+**Use “Select” for selecting from multiple options (e.g., a dropdown menu, multiple menu items, etc.)**
+
+**Use “Select” when there’s multiple instructions separated by > (e.g., Select Preferences > Languages)**
+
+For example:
+
+- **Use**: Click **Settings** to open your user settings.
+- **Use**: Select the machines you want to register, then click **Save**.
+- **Use**: Tap the application to open it.
+- **Don’t use**: Select **Add bookmark** to save your bookmark.
+
+“Click”/”Tap” is preferred over “Select” for UI elements that are definitive. This is because “Select” has an open-ended connotation, while “Click”/”Tap” is direct and definitive. Although “Click”/”Tap” may technically be wrong in certain situations (e.g., if a user is using the mobile version of a web page on their desktop computer), it’s important to consider accessibility and the primary audience of the documentation over these edge cases. Differentiating when users may have to *select* from options or *select* more than one option can be useful for them, especially when following a longer how-to guide (e.g., Click **X**, Click **Y**, Click **Z**, Select the settings you want to apply, Click **Save**). This extra level of precision can also be helpful for non-native English users and those using translated documentation.
```

Comment:
```suggestion
“Click”/”Tap” is preferred over “Select” for UI elements that are definitive. This is because “Select” has an
open-ended connotation, while “Click”/”Tap” is direct and definitive. Although “Click”/”Tap” may
technically be wrong in certain situations (e.g., if a user is using the mobile version of a web page on
their desktop computer), it’s important to consider accessibility and the primary audience of the documentation over these edge cases. Differentiating when users may have to *select* from 
options or *select* more than one option can be useful for them, especially when following a longer
how-to guide (e.g., Click **X**, Click **Y**, Click **Z**, Select the settings you want to apply, 
Click **Save**). This extra level of precision can also be helpful for non-native English users and
those using translated documentation.
```